pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
On 29/02/2016 08:56, Berker Peksag wrote:
> In pydotorg/urls.py
> <https://github.com/python/pythondotorg/pull/894#discussion_r54382722>:
> 
>> @@ -1,71 +1,72 @@
>> -from django.conf.urls import include, url, handler404
> 
> I can fix it today :)

Please go ahead, Berker. I certainly won't get to it until late tonight
and most likely tomorrow.

Also, I had to guess somewhat as to the correct approach; the idea is to
have a page which fits the broad python.org style but without the
Twitter &c. trappings from the standard template as this contravene's
the BBC's guidelines.